content/en/user-guide/lambda-tools/hot-reloading/index.md

```diff
@@ -42,6 +42,16 @@ MacOS may prompt you to grant Docker access to your target folders.
 **Layer limit with hot reloading for layers:**
 When hot reloading is active for a Lambda layer (Pro), the function can use at most one layer.
 
+{{< callout >}}
+The file sharing mechanism in Rancher Desktop or Colima distributions requires additional configuration for Lambda to be reloaded automatically.
+
+* For Rancher Desktop it is required to set the configuration `LAMBDA_DOCKER_FLAGS=-e LOCALSTACK_FILE_WATCHER_STRATEGY=polling`.
+* In case of Colima, we recommend starting it with Virtiofs mount type like this: `colima start --vm-type vz --mount-type virtiofs`.
```

Comment:
```suggestion
* For Colima, it is required to start with the Virtiofs mount type: `colima start --vm-type vz --mount-type virtiofs`.
```
Is it required or recommended? Is it necessary to use the `virtiofs` to enable it?